src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -13524,3 +13524,111 @@ def test_aks_check_network(self, resource_group, resource_group_location):
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
+        random_name_length=17, name_prefix="clitest", location="westus2"
+    )
+    def test_aks_create_cluster_with_taints(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=90750904&view=results) to validate the change.

Reply:
Seems like the pipeline failed. The original run failed due to typo I think, I fixed it locally but I'm running into some other problems. I am investigating this now. Thanks Fuming!

Reply:
Should be fixed now, PTAL!

Reply:
Queued another [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=91026075&view=results), test passed!

Reply:
Requeued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=91160256&view=results)

Reply:
Looks like it passed, thanks Fuming!